website/src/views/pageStateHandlers/CompareSideBySidePageStateHandler.spec.ts

```diff
@@ -99,7 +99,8 @@ describe('CompareSideBySideStateHandler', () => {
         const url = new URL(
             'http://example.com/covid/compare-side-by-side?' +
                 'columns=3' +
-                '&lineage%241=B.1.1.7&date%241=Last+7+Days' +
+                '&lineage%241=B.1.1.7&advancedQuery%241=country%3DUS&date%241=Last+7+Days' +
+                '&advancedQueryVariant%241=123T' +
```

Comment:
why `advancedQueryVariant`? I didn't think we had to split the advanced queries that were related to variants from other advanced queries